server/routes/shop.js

```diff
@@ -184,4 +196,50 @@ router.post("/item/delete-image/:itemid", verify, async (req, res, next) => {
     });
 });
 
+router.put("/cover-update/:userid", verify, singleUpload, async (req, res) => {
```

Comment:
same as above, rather it be: `shops/:shop_id/cover`

Reply:
You don't need upate in the URL, since a put already means update

Reply:
if I move the path variable to the middle I have to use more variables in the string template. `${API_BEFORE}${userid}${API_AFTER}` instead of `${API}${userid}`